internal/cmd/login.go

```diff
@@ -490,17 +489,20 @@ func attemptTLSRequest(options loginCmdOptions) error {
 			TLSClientConfig: &tls.Config{RootCAs: pool, MinVersion: tls.VersionTLS12},
 		},
 	}
+
 	res, err = httpClient.Do(req)
-	urlErr := &url.Error{}
-	switch {
-	case err == nil:
+
+	if err == nil {
```

Comment:
```suggestion
	if err == nil {
```

Minor style note, I think generally guidelines say to keep the `if err` statement on the line immediately after the one that assigned the `err`, to show the close relationship.